MAUI/Calendar/headers.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: post
+title: Headers in MAUI Calendar widget | Syncfusion | Calendar
+description: Learn here all about Headers feature of Syncfusion .NET MAUI Calendar (SfCalendar) widget and more.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Headers in MAUI Calendar (SfCalendar)
+You can customize all the properties of the Header view using `HeaderView`. By using this property you can customize the Background, Height, TextFormat, TextStyle and ShowNavigationArrows in the Calendar.
```

Comment:
You can customize all the properties of the Header view using `HeaderView`. By using this property, you can customize the Background, Height, TextFormat, TextStyle, and ShowNavigationArrows in the Calendar.

Reply:
Content modified for Header